citest/json_predicate/binary_predicate.py

```diff
@@ -428,8 +458,19 @@ def __call__(self, value):
     '==', lambda a, b: a == b, operand_type=list)
 LIST_NE = StandardBinaryPredicateFactory(
     '!=', lambda a, b: a != b, operand_type=list)
+
+def lists_equivalent(a, b):
```

Comment:
That makes sense. I meant `set([1, 2, 2]) == set([1, 2])`, while `!lists_equivalent([1, 2, 2], [1, 2])`.